src/helper/cloudflare-worker-domain.ts

```diff
@@ -0,0 +1,4 @@
+export const cloudflareWorkerDomain =
```

Comment:
This value is exactly the same as `serloDomain`, is a separate cloudflareWorkerDomain necessary here?

Reply:
Thanks, that's why I did not commit it directly.
I searched for that but didn't find it…
yeah, serloDomain should be fine :)